base/array.jl

```diff
@@ -27,11 +27,12 @@ import Core: arraysize, arrayset, arrayref
 
 """
     Array{T}(dims)
+    Array{T,N}(dims)
```

Comment:
I thought about it, but it seemed clearer to just write it twice.  I would have done the same thing for `SharedArray`, but `SharedArray` has so many other arguments that writing it twice seemed to make it harder to read, because it takes a second to realize that all the other arguments are the same.